carbonfive/raygun-rails
# Upgrade to Node 16.13.0 (LTS)

Problem+Solution
=======

Node 16.13.0 is the latest LTS release. Let's use it!

https://nodejs.org/en/download/

Steps to Verify
---------------

1. Install Node 16.13.0 on your machine if you don't have it.
2. Run `bin/setup`
3. Run `bin/rake` and confirm tests and linters work
4. Run `yarn start` and confirm the app loads without errors on <http://localhost:3000>
